CircuitPython_Slider/code.py.py

```diff
@@ -0,0 +1,347 @@
+import time
+import displayio
+import terminalio
+import board
+import adafruit_imageload
+from adafruit_display_text.label import Label
+from adafruit_seesaw.seesaw import Seesaw
+from adafruit_st7735r import ST7735R
+from adafruit_featherwing import minitft_featherwing
+from adafruit_motorkit import MotorKit
+from adafruit_motor import stepper
+
+#setup stepper motor
+kit = MotorKit()
+
+#setup minitft featherwing
+minitft = minitft_featherwing.MiniTFTFeatherWing()
+
+#setup bitmap file locations
+five_minBMP = "/5min_bmp.bmp"
+ten_minBMP = "/10min_bmp.bmp"
+twenty_minBMP = "/20min_bmp.bmp"
+hourBMP = "/60min_bmp.bmp"
+runningBMP = "/camSlide_bmp.bmp"
+reverseqBMP = "/reverseQ_bmp.bmp"
+backingUpBMP = "/backingup_bmp.bmp"
+stopBMP = "/stopping_bmp.bmp"
+
+# Release any resources currently in use for the displays
+displayio.release_displays()
+
+#variables for state machines in loop
+mode = 0
+onOff = 0
+pause = 0
+stop = 0
+z = 0
+
+#seesaw setup for minitft featherwing
```

Comment:
Seesaw or TFT setup shouldn't be necessary with the MiniTFT FeatherWing helper. That's all handled inside the library. If you need to access display, it's available from `minitft.display` after initialization.

Reply:
oh hey look at that, thank you! pushed updated code to reflect all changes discussed here in PR #969